filesystem/flysystem/FlysystemAssetStore.php

```diff
@@ -436,4 +735,91 @@ protected function getFileID($filename, $hash, $variant = null) {
 		return $fileID;
 	}
 
+	/**
+	 * Ensure each adapter re-generates its own server configuration files
+	 */
+	public static function flush() {
+		// Ensure that this instance is constructed on flush, thus forcing
+		// bootstrapping of necessary .htaccess / web.config files
+		$instance = singleton('AssetStore');
```

Comment:
It's called well after that. :) It's a part of the RequestFilter API, so it's safe now.